### PR TITLE
Configure i18n and add simplified Chinese (#99)

### PR DESCRIPTION
**Pull request by 1ilit**
Hello! I would really appreciate if someone fluent in Chinese could review the automated translations in '/src/i18n/locales/zh.js'

### CODE REVIEW

**Comment by felixsmli** [comment]
They look mostly fine, except these may be ambiguous:
> import_from_source: "从源导入"
> export_source: "导出源"

More explicit translation would be:
import_from_source: "导入源代码"
export_source: "导出为源代码"

Actually I think the original English text is a little bit ambiguous. It's not clear what "source" is on a first look. Personally I would prefer "Import from SQL" and "Export as SQL", thus "导入 SQL 源代码" and "导出为 SQL 源代码"

**Comment by Jackxwb** [comment]
> They look mostly fine, except these may be ambiguous:
> 
> > import_from_source: "从源导入"
> > export_source: "导出源"
> 
> More explicit translation would be: import_from_source: "导入源代码" export_source: "导出为源代码"
> 
> Actually I think the original English text is a little bit ambiguous. It's not clear what "source" is on a first look. Personally I would prefer "Import from SQL" and "Export as SQL", thus "导入 SQL 源代码" and "导出为 SQL 源代码"


--Translation--
Thank you for polishing the translation

The translation JSON file for the menu section is built using automated scripts, and the polishing and proofreading sections may not be sufficient.
My English proficiency is not good, and coupled with some missing associations between the direct translation of some menus and actual functions, there may be some shortcomings in some translations
such as
Panning

I still don't know what the purpose of this menu is (laughs)

--- 原文 ---
谢谢你对翻译的润色
﻿
菜单部分的翻译JSON文件是使用自动化脚本构建的，润色、校对部分可能有些不太够。
我的英文水平不好，加上部分菜单的直接翻译与实际功能有些缺失联想，可能部分翻译存在欠缺
比如

Settings > panning
﻿
我现在仍未知道这个菜单的作用是什么（笑）

File: src/i18n/locales/zh.js [hunk 1]
```diff
@@ -0,0 +1,213 @@
...
+  code: "zh",
+};
+
+const zh = {
+  translation: {
+    report_bug: "报告问题",
+    import: "导入",
+    file: "文件",
+    new: "新建",
+    new_window: "新窗口",
```
Comment (l123wx) [inline src/i18n/locales/zh.js:13]: 新建窗口

File: src/i18n/locales/zh.js [hunk 2]
```diff
@@ -0,0 +1,213 @@
...
+    fullscreen: "全屏",
+    settings: "设置",
+    show_timeline: "显示时间轴",
+    autosave: "自动保存",
+    panning: "平移",
+    table_width: "表格宽度",
+    language: "语言",
+    flush_storage: "清除存储",
+    are_you_sure_flush_storage:
+      "确定要清除存储吗？这将不可逆地删除所有您的图表和自定义模板。",
```
Comment (l123wx) [inline src/i18n/locales/zh.js:64]:
```suggestion
      "您确定要清除存储吗？此操作将无法恢复地删除您所有的图表和自定义模板。",
```

File: src/i18n/locales/zh.js [hunk 3]
```diff
@@ -0,0 +1,213 @@
...
+    storage_flushed: "存储已清除",
```
Comment (l123wx) [inline src/i18n/locales/zh.js:65]:
```suggestion
    storage_flushed: "存储已清空",
```

File: src/i18n/locales/zh.js [hunk 4]
```diff
@@ -0,0 +1,213 @@
...
+    help: "帮助",
+    shortcuts: "快捷键",
+    ask_on_discord: "在 Discord 上问我们",
```
Comment (l123wx) [inline src/i18n/locales/zh.js:68]:
```suggestion
    ask_on_discord: "在 Discord 联系我们",
```

File: src/i18n/locales/zh.js [hunk 5]
```diff
@@ -0,0 +1,213 @@
...
+    add_note: "添加注释",
+    add_type: "添加类型",
+    to_do: "待办事项",
+    tables: "表",
+    relationships: "关系",
+    subject_areas: "主题区域",
+    notes: "注释",
+    types: "类型",
+    search: "搜索...",
+    no_tables: "没有表",
```
Comment (l123wx) [inline src/i18n/locales/zh.js:88]:
```suggestion
    no_tables: "空空如也",
```

"空空如也" is an idiom meaning completely empty, devoid of anything.

You can change the empty prompts for "relationship," "area," "notes," etc. to this.

File: src/i18n/locales/zh.js [hunk 6]
```diff
@@ -0,0 +1,213 @@
...
+    no_issues: "未检测到问题。",
+    strict_mode_is_on_no_issues: "严格模式已关闭，因此不会显示任何问题。",
+    name: "名称",
+    type: "类型",
+    null: "空",
+    not_null: "非空",
+    primary: "主键",
+    unique: "唯一",
+    autoincrement: "自增",
+    default_value: "默认",
```
Comment (l123wx) [inline src/i18n/locales/zh.js:107]:
```suggestion
    default_value: "默认值",
```

**Review by photowey — COMMENTED** [review]
Added some small comments to the translation.

**Comment by 1ilit** [comment]
Thank you everyone!

**Comment by grinny58** [comment]
Pc wont turn on

**Pull request merged**